src/nunavut/lang/__init__.py

```diff
@@ -220,6 +222,13 @@ def set_target_language(self, target_language: typing.Optional[str]) -> "Languag
             self._target_language_name = LanguageClassLoader.to_language_name(target_language)
         return self
 
+    def load_default_config(self, language_standard: str) -> None:
+        self._ln_loader.config  # Accessing this property causes the defaults to load
+        defaults_key = f"{language_standard}_options"
+        if defaults_key in self._ln_loader.config.sections()[NUNAVUT_LANG_CPP]:
```

Comment:
This is a leaky abstraction. `nunavut.lang` should have no specific knowledge of `nunavut.lang.cpp`